src/Detector/DetectorLibrary/DetectorOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.Detector
+{
+    /// <summary>
+    /// Options to create a detector.
+    /// </summary>
+    public class DetectorOptions
+    {
+        /// <summary>
+        /// Gets or sets the source repo where the application is stored.
+        /// </summary>
+        public string SourceDir { get; set; }
+
+        public string PlatformName { get; set; }
```

Comment:
Do we need properties from line 18 to 29?

Reply:
True we don't need them anymore.